pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "MoUSE"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+librosa = "0.10.1"
+torch = "2.2.2"
+torchaudio = "2.2.2"
+torchvision = "0.17.2"
+numpy = "1.26.4"
+matplotlib = "3.8.4"
+opencv-python = "4.9.0.80"
+pandas = "2.2.2"
+scikit-image = "0.23.1"
+scikit-learn = "1.4.2"
+scipy = "1.13.0"
+pytorch_lightning = "2.2.2"
+rtree = "1.2.0"
+environ-config = "23.2.0"
+ray = "2.10.0"
+protobuf3 = "0.2.1"
+bayesian-optimization = "1.4.3"
+seaborn = "0.13.2"
```

Comment:
why pytest is missing ?
some linter should be configured as well, like yapf

Reply:
done